vhdbuilder/packer/install-dependencies.sh

```diff
@@ -443,6 +444,22 @@ for KUBERNETES_VERSION in ${PATCHED_HYPERKUBE_IMAGES}; do
   CONTAINER_IMAGE="mcr.microsoft.com/oss/kubernetes/hyperkube:v${KUBERNETES_VERSION}"
```

Comment:
so this will pull hyperkube image always, it looks like should be moved to an `else`

Reply:
+1